blocks/core/ff_module/ff_module-filter/ff_module-filter.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+$(function(){ 
+	$("[data-control='filter-dropdown']").click(function(){
+		var filterDropdown = $(this),
+		      filterIcon = filterDropdown.find("[data-icon='filter-dropdown-icon']"),
+		      filterContent = filterDropdown.next();
```

Comment:
@dkhuntrods agree. It's worth having a chat with Michael as he and I had a chat around naming of `data` elements on Friday. He's got some good ideas on this.